adalflow/adalflow/components/model_client/lm_studio_client.py

```diff
@@ -0,0 +1,120 @@
+import os
+import re
+from typing import Dict, Optional, Any, List, Union, Sequence
+import logging
+import backoff
+import requests
+from adalflow.core.types import ModelType, GeneratorOutput, EmbedderOutput, Embedding, Usage
+from adalflow.core.model_client import ModelClient
+
+log = logging.getLogger(__name__)
+
+class LMStudioClient(ModelClient):
+    """A component wrapper for the LM Studio API client."""
```

Comment:
Please add relevant links and instructions on how to set up the client and additionally some example in this doc_string.